components/DeveloperTools.tsx

```diff
@@ -154,7 +162,10 @@ export function DeveloperTools() {
                     );
                   }
                 } else if (isCountOnlyInsertOptions(options)) {
-                  if (metricValue === Metric.Count) {
+                  if (
+                    Constants?.expoConfig?.extra?.endtoend ||
```

Comment:
This seems cursed

Reply:
Resolved because bastian was like 
![image](https://github.com/user-attachments/assets/7ebe3e8e-ba5c-4ede-aae0-a0fd86bc4ab5)
 about it